sdk/core/azure-core/src/main/java/com/azure/core/http/rest/RestProxy.java

```diff
@@ -196,7 +196,7 @@ private Method determineResumeMethod(Method method, String resumeMethodName) {
      * @return The updated context containing the span context.
      */
     private Context startTracingSpan(Method method, Context context) {
-        String spanName = String.format("Azure.%s/%s", interfaceParser.getServiceName(), method.getName());
+        String spanName = String.format("%s/%s", interfaceParser.getServiceName(), method.getName());
```

Comment:
for the sake of having the same naming format as other azure SDKs. can we use '.' as a separator everywhere?

E.g. 'Appconfig.getKey' rather than 'Appconfig/getKey'